tutormfe/patches/openedx-lms-production-settings

```diff
@@ -1 +1,13 @@
-CORS_ORIGIN_WHITELIST.append("{{ MFE_HOST }}")
\ No newline at end of file
+{% if MFE_ACCOUNT_MFE_APP %}
+ACCOUNT_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ MFE_HOST }}/MFE_ACCOUNT_MFE_APP["name"]"
```

Comment:
@regisb I tried building the MFE in prod (locally) but `production.py` looks like this:
`ACCOUNT_MICROFRONTEND_URL = "http://apps.local.overhang.io/MFE_ACCOUNT_MFE_APP["name"]"`

But it's weird because in `development.py` it looks fine:
`ACCOUNT_MICROFRONTEND_URL = "http://apps.local.overhang.io:1997/account"`

Reply:
```suggestion
ACCOUNT_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ MFE_HOST }}/{{MFE_ACCOUNT_MFE_APP["name"]}}"
"

Reply:
Good catch, thanks!